logstash-scheduler/src/main/java/org/apache/mesos/logstash/config/LogstashConfig.java

```diff
@@ -9,9 +9,16 @@
 @Component
 @ConfigurationProperties(prefix = "logstash")
 public class LogstashConfig {
+
+    private static final String DEFAULT_EXECUTOR_IMAGE_NAME = "mesos/logstash-executor";
```

Comment:
There is really no need for these private constants. Just inline them on config parameters